### PR TITLE
OPS-3678/SQS-local-multiarch

### PR DESCRIPTION
**Pull request by nadsella**

### CODE REVIEW

**Comment by nadsella** [comment]
@biggianteye I know we discussed adding some info about building a multiarch and tagging it here but I feel like that information needs to sit in some documentation elsewhere, as it's going to be the same for other images and not just this one. I've just updated the README to add specifics about this multiarch image. Let me know if you think it would beneficial to add anything else.

File: README.md [hunk 1]
```diff
@@ -9,12 +9,20 @@ Docker image containing an SQS-compatible queue implementation ([ElasticMQ](http
...
-You can use a custom conf file as well to override the confih stored in the image.
+You can use a custom conf file as well to override the config stored in the image.
 
 ```
 docker run -p 9324:9324 -v "$PWD/elasticmq.conf:/elasticmq.conf" graze/sqs-local
 ```
 
+### Multiarch
+
+This allows you to run the image on various architectures. The recommendation would be to use this if you're trying to run thi on an Apple M1.
```
Comment (biggianteye) [inline README.md]:
```suggestion
This allows you to run the image on various architectures. The recommendation would be to use this if you're trying to run this on an Apple M1.
```

**Review by biggianteye — APPROVED** [review]
Looks fine. Would it be sensible to make multiarch the default?

**Comment by biggianteye** [comment]
> @biggianteye I know we discussed adding some info about building a multiarch and tagging it here but I feel like that information needs to sit in some documentation elsewhere, as it's going to be the same for other images and not just this one. I've just updated the README to add specifics about this multiarch image. Let me know if you think it would beneficial to add anything else.

Agreed. This makes sense.

**Pull request merged**